curriculum/challenges/english/25-front-end-development/quiz-javascript-audio-and-video/66edd3b3096349f06cf688bb.md

```diff
@@ -390,76 +390,71 @@ Enable JavaScript to synchronize multiple audio tracks.
 
 #### --answer--
 
-Enable JavaScript to generate media streams for playback.
+It lets you directly pass a user's webcam feed to a video element
 
 ### --question--
 
 #### --text--
 
-Which method is used to connect nodes in the Web Audio API?
+When is the `ended` event fired?
 
 #### --distractors--
 
-`linkTo()`
+When the media source is changed
 
 ---
 
-`attachNode()`
+When the `pause()` method is called
 
 ---
 
-`joinAudioPath()`
+When the media is manually stopped
 
 #### --answer--
 
-`connect()`
+When the end of the media has been reached
 
 ### --question--
 
 #### --text--
 
-What does the following code do?
-
-```javascript
-navigator.mediaDevices.enumerateDevices()
-  .then(devices => console.log(devices));
-```
+How is the Screen Capture API exposed?
 
 #### --distractors--
 
-Plays all available media devices.
+Through the `window.screenCapture` interface
 
 ---
 
-Stops all active media streams.
+By using the `navigator.capture` property
 
 ---
 
-Counts the number of connected displays.
+By using the `captureScreen()` method
 
 #### --answer--
 
-Lists all available media input and output devices.
+By calling the `getDisplayMedia()` method of the mediaDevices object
```

Comment:
```suggestion
By calling the `getDisplayMedia()` method.
```

I would just leave it here so it's closer length to the others. It's still accurate.